src/aggregation/factory.py

```diff
@@ -32,7 +31,7 @@ def load_checkpoint(model, checkpoint_path, strict=True):
     return incompatible_keys
 
 
-def create_model(cfg_path, pretrained=''):
+def create_model(cfg_path, clip_model, pretrained=''):
```

Comment:
instead of passing in the clip_model, hard code it to what we had in main.py and remove initializing the CLIP model in main.py

this is closer to what we're going to have in the end